client_operations_test.go

```diff
@@ -20,7 +20,7 @@ func TestCreateAccount(test *testing.T) {
 			return mockResponse(req, 201, String(`{"id": "abcd1234"}`))
 		},
 	}
-	client := scenario.MockHTTPClient()
+	client, _ := scenario.MockHTTPClient(nil)
```

Comment:
Let's perform the error checking so we don't introduce new linting issues. It's a best practice in go.

```suggestion
	client, err := scenario.MockHTTPClient(nil)
	if err != nil {
	    t.Fatalf("failed to initialize HTTP Client: %v", err)
	}
```